react/fields/MonthPicker/NativeMonthPicker/NativeMonthPicker.js

```diff
@@ -49,6 +49,7 @@ export default class NativeMonthPicker extends Component {
     super();
 
     this.handleChange = this.handleChange.bind(this);
+    this.handleBlur = this.handleBlur.bind(this);
```

Comment:
Assuming the addition of the `blur` event handler is for the resolution of the prefill bug? Do you think it's worth adding a descriptive comment as to why both `blur` and `change` are used? The main reason I say that is that they both have the same implementation.